contrib/firewalld/template.py

```diff
@@ -0,0 +1,70 @@
+pkgname = "firewalld"
+pkgver = "2.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-sysconfig", "--disable-systemd"]
+configure_gen = ["./autogen.sh"]
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "gtk-doc-tools",
+    "intltool",
+    "libxml2-progs",
+    "pkgconf",
+    "python",
+]
+makedepends = [
+    "gettext-devel",
+    "glib-devel",
+]
+depends = [
+    "python-dbus",
+    "python-gobject",
+    "python-nftables",
+]
+pkgdesc = "Stateful zone based firewall daemon with D-Bus interface"
+maintainer = "miko <mikoxyzzz@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://firewalld.org"
+source = f"https://github.com/{pkgname}/{pkgname}/releases/download/v{pkgver}/{pkgname}-{pkgver}.tar.bz2"
+sha256 = "22e3c700c2e4867796e9c22cf929cba4b2c143c8884240cfb9f3ef731366ff71"
+# tests don't work in our build env
+options = ["!check"]
+
+
+def do_prepare(self):
+    self.rm("src/icons/*/*/firewall-applet*", recursive=True, glob=True)
+
+
+def post_install(self):
+    from cbuild.util import python
+
+    python.precompile(self, "usr/lib")
+    self.mv(
+        self.destdir / "etc/modprobe.d", self.destdir / "usr/lib/modprobe.d"
+    )
+    self.rm(self.destdir / "etc/logrotate.d", recursive=True)
```

Comment:
maybe actually also patch all these out of the makefile instead of removing them, more robust that way